shortcut_composer/core_components/controllers/node_controllers.py

```diff
@@ -83,3 +83,26 @@ def set_value(self, visibility: bool) -> None:
         if self.active_node.visible != visibility:
             self.active_node.visible = visibility
             self.active_document.refresh()
+
+
+class LayerCreateWithBlendingController(NodeBasedController):
+    """
+    Creates Paint Layer with set Blending Mode.
+    """
+
+    default_value = BlendingMode.NORMAL
+
+    def get_value(self) -> BlendingMode:
+        """Get current layer blending mode."""
+        return self.active_node.blending_mode
```

Comment:
```suggestion
        raise NotImplementedError("Can't use this controller for getting value")
```
Pobieranie wartości tym kontrolerem nie ma sensu - nie możesz go użyć w żadnej innej templatce niż PieMenu (MultipleAssignment, TemporaryKey, MouseTracker).

Docelowo chyba zdecyduje się wydzielić klasy `Controller` i `Actuator`, gdzie Actuator nie posiada `get_value`, a Controller po nim dziedziczy dodając tę metodę. Wtedy Przekazując tę klasę do czegokolwiek innego niż PieMenu nie przeszłoby sprawdzania typów.

Ale na ten moment wystarczy taki wyjątek.

Reply:
A fakt. To moje przeoczenie po copy paste kodu. Poprawie :)
Podoba mi się pomysł z Actuatorem, ma to większy sens. Tym bardziej że kontrolery będą często używane do Pie Menusów.